api/controllers/v1/jobs.ts

```diff
@@ -157,21 +175,23 @@ async function prepSummaryMessage(
   jobTitle: string,
   failed = false
 ): Promise<string> {
-  // TODO: Determine why mms-docs has a special lastMessage slicing
+  const urls = extractUrlFromMessage(fullDocument);
+  let mms_urls = [null, null];
   if (repoName === 'mms-docs') {
-    lastMessage = `${lastMessage.slice(lastMessage.indexOf('mut-publish'))}\n\n${lastMessage.slice(
-      lastMessage.lastIndexOf('Summary')
-    )}`;
+    if (urls.length >= 2) {
+      mms_urls = urls.slice(-2);
+    }
+  }
```

Comment:
I'd add a comment here explaining why we need to do this for mmsdocs

alternately we could move lines 179, and 181 through 183 to line 194 and consolidate this boolean